clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/cache/CatsSearchProvider.groovy

```diff
@@ -50,17 +51,16 @@ class CatsSearchProvider implements SearchProvider, Runnable {
   private final Map<String, Template> urlMappings
   private final ProviderRegistry providerRegistry
 
-
-
-  private final AtomicReference<Map<String, Collection<String>>> cachedIdentifiersByType = new AtomicReference(
-    [:]
-  )
+  private final ConcurrentHashMap<String, Collection<String>> cachedIdentifiersByType = new ConcurrentHashMap<>()
 
   private final FiatPermissionEvaluator permissionEvaluator
   private final List<KeyParser> keyParsers
 
   private final ScheduledExecutorService scheduledExecutorService
 
+  @Value('${titus.enabled:false}')
```

Comment:
Do not use `@Value` annotations. Create a `ConfigurationProperties` class and inject that in instead.

Reply:
Also, no. Do not leak cloud providers into core code.